### DISCUSSION TITLE
使用 formily 封装自定义组件的一点困惑

### DISCUSSION THREAD

**Discussion by daijingfeng** [alibaba/formily#3193] [Q&A 2.x] [answered]
假设需要实现一套基于其他 UI 组件的自定义业务组件，比如 antd 的 [Form](https://ant.design/components/form-cn/)，本身带了一些表单功能（布局、联动、动态增减表单项等）。那么 formily 封装的 antd Form 组件，有对这些原有的功能做处理么？还是直接使用 formily 自己的实现呢？

**Comment by janryWang** [comment, marked as answer]
目前formily antd里的Form都是自己实现的，没有用到antd自身的表单管理能力
